docs/articles/licensing/index.rst

```diff
@@ -1,8 +1,18 @@
 :orphan:
 
-Licensing
+Licensing and Compliance
 =========
 
+PELUX is open source software. This means that PELUX is built on
```

Comment:
What parts of PELUX are intended to be included in this statement? We have a nomenclature that makes "PELUX" a bit ambiguous since it's an offering which is a bit abstract. The OSS parts that we usually describe in these docs are the PELUX Baseline (i.e. the yocto based linux platform), PELUX Software Factory Platform Blueprint (these docs), and the PELUX Baseline Software Factory (the reference SWF deployment that uses these docs).

There's a page for the nomenclature coming and I'm not saying all of the above should be documented here. But maybe we should state here that we mean the "Platform and the Platform Software Factory"? That should be detailed enough to understand roughly what parts we're talking about, without going all in on the nomenclature. And later we could add a reference to the nomenclature and structure descriptions.

Reply:
I think that the way this text is written, it is general enough to imply any or all parts of PELUX. And since all parts of PELUX are open source, the exact nomenclature isn't really an issue?

Perhaps an easy way to mitigate the problem is to say that "all parts of PELUX" are open source?

Reply:
Well that only applies to the baseline right? @tobsan